### ISSUE TITLE
dexmaker-mockito 1.5 release

### ISSUE THREAD

**Issue by ryansgot** [linkedin/dexmaker#32]
Looking at the code, I see that the null pointer exception in dexmaker-mockito 1.4's `DexmakerMockMaker.getInvocationHandlerAdapter` has been fixed since September (17cc9ccdd52a8b27793ec12f3cebb93ddf469af4). Is there any ETA on a release? I can build myself, but I'd prefer to not have to do so.

**Comment by vaughandroid** [comment]
This project is kind of dead. :(

The good news is, to tide things over you can use JitPack.io to get a build with the fix:

```
def dexmakerVersion = '17cc9cc'
def dexmakerGroupId = 'com.github.crittercism.dexmaker'
androidTestCompile "${dexmakerGroupId}:dexmaker:${dexmakerVersion}"
androidTestCompile "${dexmakerGroupId}:dexmaker-dx:${dexmakerVersion}"
androidTestCompile "${dexmakerGroupId}:dexmaker-mockito:${dexmakerVersion}"
```

**Comment by drewhannay** [comment]
Good news! The project isn't dead anymore 😄 

I just released (1.5.1)[https://github.com/linkedin/dexmaker/releases], which contains the fix that @ryansgot, plus some other fixes (most notably, transitive dependencies now work). Keep an eye out for Mockito 2.x support coming in the near future as well.

**Comment by ryansgot** [comment]
@drewhannay Thanks a lot. I'll start using 1.5.1.
